test/unit/tasksTest.js

```diff
@@ -380,6 +380,9 @@ describe('Rerun Task Tests', () => {
     it('will process empty tasks', async () => {
       const originalTask = await createTask()
       await tasks.findAndProcessAQueuedTask()
+      await new Promise(resolve => {
```

Comment:
Is this basically just a sleep? Shouldn't we await the actual execution somehow instead of giving a hardcoded time?